dlpy/utils.py

```diff
@@ -1339,6 +1460,117 @@ def create_object_detection_table(conn, data_path, coord_type, output,
     A list of variables that are the labels of the object detection table
 
     '''
+
+    return create_table_based_on_pascal_voc_format(conn, data_path, coord_type, output,
+                                                   local_path, image_size, task='object detection')
+
+
+def create_instance_segmentation_table(conn, data_path, coord_type, output, local_path=None, image_size=(416, 416)):
+    '''
+    Create an instance segmentation table
+
+    Parameters
+    ----------
+    conn : session
+        CAS connection object
+    data_path : string
+        Specifies a location where annotation files and image files are stored.
+        Annotation files should be XML file based on Pascal VOC format
+        Notice that the path should be accessible by CAS server.
+    coord_type : string
+        Specifies the type of coordinate to convert into.
+        'yolo' specifies x, y, width and height, x, y is the center
+        location of the object in the grid cell. x, y, are between 0
+        and 1 and are relative to that grid cell. x, y = 0,0 corresponds
+        to the top left pixel of the grid cell.
+        'coco' specifies xmin, ymin, xmax, ymax that are borders of a
+        bounding boxes.
+        The values are relative to parameter image_size.
+        Valid Values: yolo, coco
+    output : string
+        Specifies the name of the instance segmentation table.
+    local_path : string, optional
+        Local_path and data_path point to the same location.
+        The parameter local_path will be optional (default=None) if the
+        Python client has the same OS as CAS server or annotation files
+        in TXT format are placed in data_path.
+        Otherwise, the path that depends on the Python client OS needs to be specified.
+        For example:
+        Windows client with linux CAS server:
+        data_path=/path/to/data/path
+        local_path=\\path\to\data\path
+        Linux clients with Windows CAS Server:
+        data_path=\\path\to\data\path
+        local_path=/path/to/data/path
+    image_size : tuple or integer, optional
+        Specifies the size of images to resize.
+        If a tuple is passed, the first integer is width and the second value is height.
+        Default: (416, 416)
+
+    Returns
+    -------
+    A list of variables that are the labels of the object detection task and the label of instance segmentation task
+
+    '''
+
+    return create_table_based_on_pascal_voc_format(conn, data_path, coord_type, output,
```

Comment:
if we only return voc pascal format, we have to ask format type and call this function based on that.

Reply:
I think the function name makes you confused. It should be create_table_from_pascal_voc_format() which using pascal voc annotation files to create the table.